idaes/models_extra/co2_capture_and_utilization/unit_models/__init__.py

```diff
@@ -0,0 +1,13 @@
+#################################################################################
+# The Institute for the Design of Advanced Energy Systems Integrated Platform
+# Framework (IDAES IP) was produced under the DOE Institute for the
+# Design of Advanced Energy Systems (IDAES).
+#
+# Copyright (c) 2018-2023 by the software owners: The Regents of the
```

Comment:
Copyright year range should be: 2018-2024

Reply:
Copyright still needs to be updated.